xla/service/gpu/cudnn_fused_mha_rewriter.cc

```diff
@@ -358,67 +360,86 @@ StatusOr<bool> IsSupportedBMM2(const HloInstruction* bmm_2,
   return true;
 }
 
-bool MatchDefaultBmmBmm(int64_t bmm2_operand_position, HloInstruction* instr,
-                        HloInstruction** bmm_1, HloInstruction** bmm_2,
-                        std::string& custom_call_name) {
+bool MatchDefaultFwdBmmBmm(int64_t bmm2_operand_position, HloInstruction* instr,
```

Comment:
Output parameters aren't great, consider returning a struct?

Reply:
refactored all output parameters to be returned from struct for both forward and backward calls